contents/session-replay.mdx

```diff
@@ -0,0 +1,159 @@
+---
+title: Session replay
+subtitle: <span class="text-red font-semibold bg-yellow/10">Watch recordings of visitors</span> using your product or website.
+featuredImage: ./images/products/session-recording.png
+template: product
+productFeatures: [
+    {
+        icon: 'Actions',
+        title: 'Capture sessions without extra code',
+        description: 'Works with PostHog.js',
+    },
+    {
+        icon: 'Playlist',
+        title: 'Automatic playlists',
+        description: 'Filter by user behavior or time',
+    },
+    {
+        icon: 'Mobile',
+        title: 'Mobile session recording',
+        description: 'iOS library now in beta',
```

Comment:
```suggestion
        description: 'Web or iOS (beta) available',
```

Reply:
maybe icon here ins't perfect. i'm worried it could read that we _only_ do mobile the way it is at the moment